docs/state-sync.md

```diff
@@ -28,6 +28,13 @@ If `$PIO_HOME/data/` contains prior data, you must first clean it up in order fo
 ### Mainnet
 
 ```  
+# Initialization is required ONLY on first time setups. Otherwise, skip this step.
+mkdir ~/.provenanced
+export PIO_HOME=~/.provenanced
+provenanced init choose-a-moniker --chain-id pio-mainnet-1
+curl https://raw.githubusercontent.com/provenance-io/mainnet/main/pio-mainnet-1/genesis.json > genesis.json
+mv genesis.json $PIO_HOME/config
```

Comment:
Is there a reason to not just output this to the location it should be in right off the bat?
```suggestion
curl https://raw.githubusercontent.com/provenance-io/mainnet/main/pio-mainnet-1/genesis.json > "$PIO_HOME/config/genesis.json"
```
Same with below.

Reply:
No. I copied that portion of the instructions from [running-a-mainnet-node](https://docs.provenance.io/blockchain/running-a-node/running-a-node-1/join-provenance-testnet/running-a-mainnet-node).

I like one liners as well.